easybuild/easyconfigs/l/libjpeg-turbo/libjpeg-turbo-1.3.0-goolf-1.4.10.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'libjpeg-turbo'
+version = '1.3.0'
+
+homepage = 'http://sourceforge.net/libjpeg-turbo/'
+description = """ libjpeg-turbo is a fork of the original IJG libjpeg which 
+ uses SIMD to accelerate baseline JPEG compression and decompression. libjpeg 
+ is a library that implements JPEG image encoding, decoding and transcoding.
+"""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
```

Comment:
@pescobar existing easyconfigs for `libjpeg-turbo` have `toolchainopts = {'pic': True}` here, and also include custom `sanity_check_paths`, please do that here too for consistency?